[zod-openapi] Cannot find module '@asteasolutions/zod-to-openapi/dist/v3.0/openapi-generator' on NodeNext and ES Modules

My environment is
* @hono/zod-to-openapi@0.14.9
* hono@4.4.13
* typescript@5.5.3
* zod@3.23.8

I ran `tsc` following TypeScript.

```ts
import { z } from '@hono/zod-openapi'
```

Then, I got the following error

```console
$ npx tsc
node_modules/@hono/zod-openapi/dist/index.d.mts:5:37 - error TS2307: Cannot find module '@asteasolutions/zod-to-openapi/dist/v3.0/openapi-generator' or its corresponding type dec
larations.                                                                                                                                                                        
5 import { OpenAPIObjectConfig } from '@asteasolutions/zod-to-openapi/dist/v3.0/openapi-generator';
                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/@hono/zod-openapi/dist/index.d.mts:5


```
tsconfig.json
```json
{
  "compilerOptions": {
    "module": "NodeNext",
    "moduleResolution": "NodeNext",
    "skipLibCheck": false
  }
}
```
package.json
```json
{
  "dependencies": {
    "@hono/zod-openapi": "^0.14.9",
    "hono": "^4.413",
    "zod": "^3.23.8"
  },
  "devDependencies": {
    "typescript": "^5.5.3"
  },
  "type": "module"
}
```

--- a/.changeset/unlucky-cheetahs-fry.md
+++ b/.changeset/unlucky-cheetahs-fry.md
@@ -1,0 +1,5 @@
+---
+'@hono/zod-openapi': minor
+---
+
+fix(zod-openapi): infer OpenAPIObjectConfig

--- a/.changeset/unlucky-cheetahs-fry.md
+++ b/.changeset/unlucky-cheetahs-fry.md
@@ -1,5 +1,5 @@
 ---
-'@hono/zod-openapi': minor
+'@hono/zod-openapi': patch
 ---
 
 fix(zod-openapi): infer OpenAPIObjectConfig

--- a/packages/zod-openapi/src/index.ts
+++ b/packages/zod-openapi/src/index.ts
@@ -12,7 +12,6 @@ import {
   OpenApiGeneratorV31,
   extendZodWithOpenApi,
 } from '@asteasolutions/zod-to-openapi'
-import type { OpenAPIObjectConfig } from '@asteasolutions/zod-to-openapi/dist/v3.0/openapi-generator'
 import { zValidator } from '@hono/zod-validator'
 import { Hono } from 'hono'
 import type {
@@ -251,6 +250,10 @@ export type RouteHook<
   P,
   RouteConfigToTypedResponse<R> | Response | Promise<Response> | void | Promise<void>
 >
+
+type OpenAPIObjectConfig = Parameters<
+  InstanceType<typeof OpenApiGeneratorV3>['generateDocument']
+>[0]
 
 export type OpenAPIObjectConfigure<E extends Env, P extends string> =
   | OpenAPIObjectConfig

